src/main/java/com/purbon/kafka/topology/TopologyValidator.java

```diff
@@ -76,33 +76,53 @@ public TopologyValidator(TopologyBuilderConfig config) {
         .collect(Collectors.toList());
   }
 
-  class VoidValidation implements TopologyValidation {
-    @Override
-    public void valid(Topology topology) {}
-  }
-
   private List<Validation> validations() {
     return config.getTopologyValidations().stream()
         .map(
             validationClass -> {
               try {
-                String fullValidationClass = classPrefix + validationClass;
-                Class<?> clazz = Class.forName(fullValidationClass);
+                Class<?> clazz;
+                clazz = getValidationClazz(validationClass);
+                if (clazz == null) {
+                  String deprecatedValidationClassConfig = classPrefix + validationClass;
```

Comment:
would this instead of a config be a namespace or full qualified class name? config looks strange 😄

Reply:
Agree, fixed that.